.git-blame-ignore-revs

```diff
@@ -12,5 +12,6 @@ ed6cdae66d80d46c74b8f29d14c488831d5d6e5c
 81c0f8bf5494e8c92dce72e4baf98a2e8bf5906b
 # Split models.py into multiple files
 2b88cfc3d827781444f3f8f79b1bcc16d72fc486
-# Add types to grpc servicers
+# Add types
 055ef4610f37155d31aa98f8047c329fab250d8f
+10ffc6710ca1c5e59144765c21d01df13681ad30
```

Comment:
this is the wrong commit now? I think you don't need to include this. and please only include stuff that contains entirely mechanical work (e.g. type annotations/formatting/etc), but if you in the same commit also do any even slightly real changes, let's try to keep it out of here

Reply:
Very good point. Removed the commit.